Userland/Utilities/netstat.cpp

```diff
@@ -89,7 +124,20 @@ int main(int argc, char** argv)
         }
     };
 
+    auto get_formatted_program = [&](pid_t pid) {
+        auto program = programs.get(pid);
+        if (current_uid != 0 && current_uid != program.value().uid) {
```

Comment:
This check should be done in the kern and not in the userland program as a user could simply consult /proc/net themselves, circumventing the check.

Reply:
Yep. You're right -- fixed. Thanks!